Pkgfile

```diff
@@ -164,9 +164,9 @@ vars:
   arm_trusted_firmware_sha512: 958cf8f9e258638e59d0fbd8b053fce7d8a9ea2fc922686c9d20ea16f79f55219ac18a12ab240c528ee98e49c2e0eef4c963fdb255cc14b92437a5b3cffc8640
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
-  uboot_version: 2022.07
```

Comment:
@frezbo is working on u-boot update, it's not that simple as bumping a version

Reply:
got it, reverted this one.